build.sbt

```diff
@@ -70,22 +70,32 @@ lazy val portal = project
   )
 
 lazy val utils =
-  (project in file("utils")).settings(libraryDependencies ++= jsonDependencies ++ testContainers)
+  (project in file("utils"))
+    .configs(IntegrationTest)
+    .settings(
+      libraryDependencies ++= jsonDependencies ++ testContainers ++ testDependencies :+ configHokon :+ playFunctional,
+      commonSettings
+    )
+addCommandAlias("testAll", "; test ; it:test")
```

Comment:
Is it supposed to be a part of `utils` settings? Because now it is not part of it.
Also, I really doubt that we will ever create the integration tests for the `utils` module.

Reply:
1) fixed 
2) testContainers and .configs(IntegrationTest) need for TestContainersUtils, testDependencies for AuthUtilsTest, redundant alias removed